masterlist.yaml

```diff
@@ -53,6 +53,8 @@ common:
       text: 'Borrar. Esta incluido, o de otra modo, arreglado en el %1%.'
     - lang: ru
       text: 'Удалите. Уже включено в %1% или исправлено иным образом.'
+      - lang: sv
+        text: 'Ta bort. Redan inkluderat i eller fixat i %1%'
```

Comment:
Indentation is off by two spaces.
![image](https://user-images.githubusercontent.com/1944639/67122703-4ee65200-f1e6-11e9-81b5-8ca11ce85657.png)